onnx/onnx.in.proto

```diff
@@ -430,8 +487,22 @@ message ModelProto {
   // One FunctionProto can reference other FunctionProto in the model, however, recursive reference
   // is not allowed.
   repeated FunctionProto functions = 25;
+
+  // Describes different target configurations for a multi-device use case.
+  // A model can describe multiple multi-device configurations for execution.
+  repeated ConfigurationProto configuration = 26;
 };
 
+// ConfigurationProto describes a multi-device configuration for a model.
+message ConfigurationProto {
```

Comment:
Consideration on naming? Was there thoughts on calling it DeviceConfigurationProto?